deeppavlov/metrics/fmeasure.py

```diff
@@ -393,3 +393,83 @@ def precision_recall_f1(y_true, y_pred, print_results=True, short_report=False,
                 tot_predicted=results[entity_of_interest]['n_pred'])
         log.debug(s)
     return results
+
+
+def get_chunks(labels):
+    chunks = []
+    chunk_start, chunk_type = None, None
+    tmp = labels + ["O"]
+    for i, label in enumerate(tmp):
+        # end of the prev. chunk
+        if label == "O" or label == "<PAD>" or label == "<UNK>":
+            if chunk_type is not None:
+                chunks.append((chunk_type, chunk_start, i))
+                chunk_start, chunk_type = None, None
+        else:
+            prefix, suffix = label.split('-')
+            # end of the prev. chunk and the current position is a single chunk
+            if prefix == "S":
+                if chunk_type is not None:
+                    chunks.append((chunk_type, chunk_start, i))
+                chunks.append((suffix, i, i + 1))
+                chunk_start, chunk_type = None, None
+            # start a new chunk
+            elif prefix == "B":
+                if chunk_type is not None:
+                    chunks.append((chunk_type, chunk_start, i))
+                chunk_start, chunk_type = i, suffix
+
+    return chunks
+
+
+def print_prf(p, r, f):
+    s = "\n"
+    for chunk_name in p:
+        s += f"{chunk_name:12} : {p[chunk_name] * 100:6.2f} | {r[chunk_name] * 100:6.2f} | {f[chunk_name] * 100:6.2f}\n"
+    log.debug(s)
+
+
+@register_metric('ner_prf1')
+def ner_prf1(batch_labels, batch_preds):
```

Comment:
Please add a [Google Style docstring](http://google.github.io/styleguide/pyguide.html#381-docstrings) describing what this metric is in a couple of words

Reply:
added the function description